README.md

```diff
@@ -52,7 +55,8 @@ var api = new ParseServer({
   cloud: '/home/myApp/cloud/main.js', // Provide an absolute path
   appId: 'myAppId',
   masterKey: 'mySecretMasterKey',
-  fileKey: 'optionalFileKey'
+  fileKey: 'optionalFileKey',
+  serverURL: 'http://localhost:' + port // Don't forget to change to https if needed
```

Comment:
this would also need the /parse or whatever the mount point is.  For this readme, I'd include a full example url, like `http://localhost:1337/parse`